.circleci/continue_config.yml

```diff
@@ -52,7 +52,31 @@ jobs:
       - store_artifacts:
           path: common/build/reports/tests
 
-  validate-search:
+  validate-search-check:
+    parameters:
+      project:
+        type: string
+    executor:
+      name: hmpps/java
+      tag: "21.0"
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gradle-{{ checksum "<< parameters.project >>/build.gradle.kts" }}
+            - gradle-
+      - run:
+          command: ./gradlew :<< parameters.project >>:check -x test
```

Comment:
When running the `check` task ignore the tests - we'll run them separately.

Reply:
I find this confusing - there is a `validate-search-check` task that doesn't actually run all the checks?  What does it do then?